subgraphs/apeswap/config/bsc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "network": "bsc",
+    "address": "0x0841BD0B734E4F5853f0dD8d7Ea041c241fb0Da6",
+    "startBlock": 4855900,
+    "yield_address": "0x5c8D727b265DBAfaba67E050f2f739cAeEB4A6F9",
+    "abi": "Yield",
```

Comment:
Change `Yield` to `MasterChef` (also other places)

Reply:
Fixed